src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceDocTransformer.java

```diff
@@ -26,20 +29,29 @@
 import com.google.api.codegen.viewmodel.GrpcDocView;
 import com.google.api.codegen.viewmodel.ImportSectionView;
 import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.api.codegen.viewmodel.metadata.ModuleView;
+import com.google.api.codegen.viewmodel.metadata.SimpleModuleView;
+import com.google.api.codegen.viewmodel.metadata.TocContentView;
+import com.google.api.codegen.viewmodel.metadata.TocModuleView;
+import com.google.api.tools.framework.model.Interface;
 import com.google.api.tools.framework.model.Model;
 import com.google.api.tools.framework.model.ProtoFile;
 import com.google.common.collect.ImmutableList;
 import java.util.List;
 
 public class RubyGapicSurfaceDocTransformer implements ModelToViewTransformer {
+  private static final int VERSION_MODULE_RINDEX = 1;
```

Comment:
Ideally this constant should not be defined twice (here and in `RubyGapicSurfaceTransformer`). Is there a common location it can be shared from?

Reply:
Done